docs/configuration/backends/docker.md

```diff
@@ -145,6 +145,18 @@ To enable constraints see [backend-specific constraints section](/configuration/
 
 ## Labels: overriding default behaviour
 
+!!! note
+    If you use a docker-compose file, labels should be defined in the `deploy` part of your service.
+
+```yaml
+version: "3"
+services:
+  nginx:
```

Comment:
Can you use another service name? Like `whoami` for example? :wink:

Reply:
Oups sure, I always use nginx as service example, I didn't realize.